### PR TITLE
feat: :sparkles: create top progress with gradient color

### PR DESCRIPTION
**Pull request by sakilk130**
Resolved: #45 

- Top progress bar with gradient color

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import 'focus-visible';
-
+import 'nprogress/nprogress.css';
```
Comment (mimukit) [inline src/pages/_app.tsx:2]: Move this CSS file import to the CSS import block below:

```diff
import 'focus-visible';
- import 'nprogress/nprogress.css';
import Head from 'next/head';
import superjson from 'superjson';
import { Baloo_Da_2 } from '@next/font/google';

+ import 'nprogress/nprogress.css';
import '../styles/globals.css';
import '../styles/tailwind.css';
```

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ import { getBaseUrl } from '../utils/getBaseUrl';
 import { getPageTitle } from '../utils/getPageTitle';
 import { getNavigationItems } from '../utils/getNavigationItems';
 import { BlogLayout } from '../layouts/BlogLayout';
+import ProgressBar from '../components/ProgressBar';
```
Comment (mimukit) [inline src/pages/_app.tsx:30]: Update the import to the named import:

```diff
- import ProgressBar from '../components/ProgressBar';
+ import { ProgressBar } from '../components/ProgressBar';
```

**Review by mimukit — CHANGES_REQUESTED** [review]
Great work @sakilk130. A couple of small changes are requested.

**Comment by sakilk130** [comment]
Thank you @mimukit Bhai for your review.
I already fix those little changes. Please have a look.

**Review by mimukit — APPROVED** [review]

**Pull request merged**